### DISCUSSION TITLE
AR Resolver version

### DISCUSSION THREAD

**Discussion by wandth** [Autodesk/maya-usd#2735] [Q&A] [answered]
Did maya-usd 0.20 still use Ar Resolver 1.0 in windows platform?

**Comment by tanant** [comment, marked as answer]
Yes - it's mainly depending on the USD version rather than the MayaUSD version though..

In detail: existing MayaUSD binary releases are built against the USD we stabilized against for that Maya version (see: https://knowledge.autodesk.com/support/maya/learn-explore/caas/simplecontent/content/maya-usd-bifrost-and-arnold-compatibility.html). 

For USD 2**1**.11 based releases, we're on AR 1.0

For USD 2**2**.x (assume .11 as the most recently released version) we should be moving to AR 2.0 

(and we don't have any variation across platforms)
